angular/demo/package.json

```diff
@@ -4,13 +4,54 @@
 	"type": "module",
 	"scripts": {
 		"ng": "ng",
-		"dev": "ng serve",
-		"dev:coverage": "node --import @agnos-ui/code-coverage/interceptReadFile ../../node_modules/@angular/cli/bin/ng.js serve",
-		"build": "ng build",
-		"build:coverage": "node --import @agnos-ui/code-coverage/interceptReadFile ../../node_modules/@angular/cli/bin/ng.js build --configuration=coverage",
+		"dev": "wireit",
+		"dev:coverage": "wireit",
+		"build": "wireit",
+		"build:coverage": "wireit",
 		"watch": "ng build --watch --configuration development",
 		"preview": "node scripts/preview.cjs dist --port 4200 --single"
 	},
+	"wireit": {
+		"dev": {
+			"command": "ng serve",
+			"dependencies": [
```

Comment:
As discussed, let's add a dependency on build (because of the issue with Angular).